### PR TITLE
Update multicell migration guide

### PR DESCRIPTION
**Pull request by Topherhindman**
## What are you changing in this pull request and why?

Changing the link for accessing dbt cloud in the migration guide to point back to our normal docs site from next.docs.getdbt.com

This also adds a blurb about login with github not being supported in multi cell. 

Screenshot 2024-02-16 at 7 29 01 AM

Resolves [MUL-642](https://dbtlabs.atlassian.net/browse/MUL-642)
Resolves [MUL-542](https://dbtlabs.atlassian.net/browse/MUL-542)
## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

[MUL-642]: https://dbtlabs.atlassian.net/browse/MUL-642?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[MUL-542]: https://dbtlabs.atlassian.net/browse/MUL-542?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/docs/docs/cloud/migration.md [hunk 1]
```diff
@@ -34,4 +34,7 @@ After migration, if you completed all the [Pre-migration checklist](#pre-migrati
 
 You have the option to log in to dbt Cloud at a different URL:
  * If you were previously logging in at `cloud.getdbt.com`, you should instead plan to login at `us1.dbt.com`. The original URL will still work, but you’ll have to click through to be redirected upon login.
- * You may also log in directly with your account’s unique [access URL](https://next.docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account).
+ * You may also log in directly with your account’s unique [access URL](https://docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account)
```
Comment (mirnawong1) [inline website/docs/docs/cloud/migration.md]:
```suggestion
 * You may also log in directly with your account’s unique [access URL](/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account)
```

File: website/docs/docs/cloud/migration.md [hunk 2]
```diff
@@ -34,4 +34,7 @@ After migration, if you completed all the [Pre-migration checklist](#pre-migrati
 
 You have the option to log in to dbt Cloud at a different URL:
  * If you were previously logging in at `cloud.getdbt.com`, you should instead plan to login at `us1.dbt.com`. The original URL will still work, but you’ll have to click through to be redirected upon login.
- * You may also log in directly with your account’s unique [access URL](https://next.docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account).
+ * You may also log in directly with your account’s unique [access URL](/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account)
+
+:::info Login with GitHub
+Users who previously utilized the "Login with GitHub" functionality will no longer be able to use this method to login to dbt Cloud after migration. To continue accessing your account, you can use your existing email and password.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/migration.md]:
```suggestion
Users who previously used the "Login with GitHub" functionality will no longer be able to use this method to login to dbt Cloud after migration. To continue accessing your account, you can use your existing email and password.
```

**Review by mirnawong1 — APPROVED** [review]
lgtm, thank you @Topherhindman and made one small tweak (utilized -> used). Approving and merging for you ✨!

File: website/docs/docs/cloud/migration.md [hunk 3]
```diff
@@ -34,4 +34,7 @@ After migration, if you completed all the [Pre-migration checklist](#pre-migrati
 
 You have the option to log in to dbt Cloud at a different URL:
  * If you were previously logging in at `cloud.getdbt.com`, you should instead plan to login at `us1.dbt.com`. The original URL will still work, but you’ll have to click through to be redirected upon login.
- * You may also log in directly with your account’s unique [access URL](https://next.docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account).
+ * You may also log in directly with your account’s unique [access URL](/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account)
+
+:::info Login with GitHub
+Users who previously used the "Login with GitHub" functionality will no longer be able to use this method to login to dbt Cloud after migration. To continue accessing your account, you can use your existing email and password.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/migration.md:40]: also suggesting the below so it's consistent with 2nd person but not tied to it 

```suggestion
If you previously used the "Login with GitHub" functionality, you won't be able to use this method to login to dbt Cloud after migration. You can use your existing email and password to continue accessing your account.
```

File: website/docs/docs/cloud/migration.md [hunk 4]
```diff
@@ -34,4 +34,7 @@ After migration, if you completed all the [Pre-migration checklist](#pre-migrati
 
 You have the option to log in to dbt Cloud at a different URL:
  * If you were previously logging in at `cloud.getdbt.com`, you should instead plan to login at `us1.dbt.com`. The original URL will still work, but you’ll have to click through to be redirected upon login.
- * You may also log in directly with your account’s unique [access URL](https://next.docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account).
+ * You may also log in directly with your account’s unique [access URL](/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account)
```
Comment (matthewshaver) [inline website/docs/docs/cloud/migration.md]:
```suggestion
 * You may also log in directly with your account’s unique [access URL](/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account).
```

**Pull request merged**